lib/common/api/deprecate.ts

```diff
@@ -34,9 +34,25 @@ const deprecate: ElectronInternal.DeprecationUtil = {
     }
   },
 
+  // remove a function with no replacement
+  removeFunction: (fn, removedName) => {
+    // if the function has already been removed, warn about it
+    if (!fn) {
+      deprecate.log(`Unable to remove function '${removedName}' from an object that lacks it.`)
```

Comment:
We can remove this:
```
deprecate.log(`Unable to remove function '${removedName}' from an object that lacks it.`)
```
since it wouldn't apply to this.

Reply:
But we keep this notice on `removeProperty`?